### PR TITLE
Fix docstring

### PR DESCRIPTION
**Pull request by sofianehaddad**
The PR fixes "the the" in several docstrings

### CODE REVIEW

File: python/src/Wilks_doc.i.in [hunk 1]
```diff
@@ -16,7 +16,7 @@ the minimal sample size :math:`N_{\alpha, \beta, i}` to perform in order to
 garantee that the empirical quantile :math:`\alpha`, noted
 :math:`\tilde{q}_{\alpha} N_{\alpha, \beta, i}` evaluated with the
 :math:`(n - i)^{th}` maximum of the sample, noted :math:`X_{n - i}` be greater
-than the theoretical quantile :math:`q_{\alpha}` with a probability at least
```
Comment (dbarbier) [inline python/src/Wilks_doc.i.in]: This one is a false positive

**Comment by adutfoy** [comment]
+1

**Comment by regislebrun** [comment]
+1

**Comment by jschueller** [comment]
+1

**Pull request merged**